cmd/execute.go

```diff
@@ -105,21 +110,21 @@ func executeTransformers() {
 	if len(ethStorageInitializers) > 0 {
 		newDiffStorageHealthCheckMessage := []byte("storage watcher for new diffs starting\n")
 		newDiffStatusWriter := fs.NewStatusWriter(healthCheckFile, newDiffStorageHealthCheckMessage)
-		newDiffStorageWatcher := watcher.NewStorageWatcher(&db, newDiffBlockFromHeadOfChain, newDiffStatusWriter)
+		newDiffStorageWatcher := watcher.NewStorageWatcher(&db, newDiffBlockFromHeadOfChain, newDiffStatusWriter, minTimeBetweenTransforms)
```

Comment:
Hm, I really like the approach in this PR and think it's super clean, but I also really want to make exceptions for new and unrecognized diffs. Do you think it would over-complicate things to only throttle pending diffs?

Reply:
We could just make the minTime 0 for `New` and `Unrecognized`.

Reply:
Nice! Yeah I think that would be good